docs/reference/configuration.rst

```diff
@@ -179,20 +179,20 @@ Full Configuration Options
                 stylesheets:
```

Comment:
Would appreciate if we can completely remove this part and replace this with one webpack_encore entrypoint.

```yaml
webpack_encore:
    builds:
        sonata_admin: '%kernel.project_dir%/public/bundles/sonataadmin

```

Reply:
IMHO this PR is big enough, and I plan to do it at some point but please, read the next steps section, there is a huge list of things to do and consider

Reply:
Just added your consideration. 

Also to be note, I took inspiration of how easy admin handles its js and css, and I didnt found the webpack encore bundle integration in place, they just generate the minifies files but include them normally.

Maybe @javiereguiluz could help us with why he took that path.

Anyway , as I said, for me this should be considered after this Pr to keep them small enough.

Reply:
@jordisala1991 I don't remember exactly why we did this 🤔  But yes, we may need to include our own styles using just Webpack Encore entries (and leave the option to include raw CSS files for those who need it). I'll add this to my TODO list. Cheers!